LayoutKitSampleApp/Benchmarks/FeedItemCellNode.swift

```diff
@@ -50,9 +51,23 @@ final class FeedItemCellNode: ASCellNode, DataBinder {
 		contentDomainNode.attributedText = NSAttributedString(string: data.contentDomain)
 		actorCommentNode.attributedText = NSAttributedString(string: data.actorComment)
 		// Pretend these are dynamic images
-		actorImageNode.image = UIImage(named: "50x50.png")
-		posterImageNode.image = UIImage(named: "50x50.png")
-		contentImageNode.image = UIImage(named: "350x200.png")
+    
+		var smallImage = FeedItemCellNode.cache.object(forKey:"50x50.png")
+		var largeImage = FeedItemCellNode.cache.object(forKey:"350x200.png")
+
+		if (smallImage == nil) {
+      smallImage = UIImage(named: "50x50.png")
```

Comment:
Sorry about the indentation, something is messed up with this file.  I couldn't get it to work properly with _either_ tabs or spaces.  Let's land it like this and clean up later :)